cfme/infrastructure/virtual_machines.py

```diff
@@ -1071,113 +1174,118 @@ class VmAllWithTemplatesDetails(CFMENavigateStep):
     VIEW = InfraVmDetailsView
     prerequisite = NavigateToSibling('All')
 
-    def step(self, *args, **kwargs):
-        sel.click(self.obj.find_quadicon(do_not_navigate=True))
+    def step(self):
+        self.prerequisite_view.sidebar.vmstemplates.tree.click_path('All VMs & Templates')
+        try:
+            row = self.prerequisite_view.entities.get_first_entity(by_name=self.obj.name)
```

Comment:
I wouldn't call it 'row' here as it may be a quadicon or tile, you never selected Grid View specifically.